sale_product_variant_attribute_tax/__manifest__.py

```diff
@@ -1,14 +1,15 @@
 # Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+# Copyright 2024 Tecnativa - Carolina Fernandez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Put attribute taxes on sales orders",
     "summary": "Bring the taxes associated to product values",
-    "version": "13.0.1.0.0",
+    "version": "16.0.1.0.0",
     "development_status": "Production/Stable",
     "category": "Sales/Sales",
     "website": "https://github.com/OCA/product-variant",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "depends": ["product_variant_attribute_tax", "sale_variant_configurator"],
+    "depends": ["sale_variant_configurator", "product_variant_attribute_tax"],
```

Comment:
Don't modify this order. It was even in alphabetical order, which is the desirable thing.